test/ResultSet/ReturnType.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+
+namespace ZendTest\Db\ResultSet;
+
+
+class ReturnType
```

Comment:
If I'm not mistaken you've created the ReturnType class to force a null to be returned from exchangeArray. You might want to add that in the class' docblock to make that clear to other developers.